advancedTodo/src/redux/todoAppComplete/todoConstants.js

```diff
@@ -1,7 +1,10 @@
 export const ADD_TODO = 'ADD_TODO';
-export const DELETE_TODO= 'DELETE_TODO';
+export const DELETE_TODO = 'DELETE_TODO';
 export const COMPLETE_TODO = 'COMPLETE_TODO';
 export const COMPLETED = 'completed';
 export const SAVE_DESCRIPTION = 'SAVE_DESCRIPTION';
-export const SHOW_DESCRIPTION= 'SHOW_DESCRIPTION';
-export const CREATE_NOTIFICATION= 'CREATE_NOTIFICATION';
\ No newline at end of file
+export const SHOW_DESCRIPTION = 'SHOW_DESCRIPTION';
+export const CREATE_NOTIFICATION = 'CREATE_NOTIFICATION';
+export const SUCCESS_MESSAGE = "Information successfully saved', 'Congratulations !"

```

Comment:
The success messages in `todoConstants.js` could be improved:

*   Instead of using a string like `"Information successfully saved', 'Congratulations !"`, it would be better to separate the message and title. This improves readability and makes it easier to modify them independently later.

*   The current message is specific to saving information. A more generic message like "Todo updated successfully" would be more appropriate since it covers saving descriptions, marking todos as complete, etc.

You can modify the messages like this:

```javascript
export const SUCCESS_MESSAGE_TITLE = 'Congratulations!';
export const SUCCESS_MESSAGE_BODY = 'Todo updated successfully';

export const DELETE_MESSAGE_TITLE = 'Congratulations!';
export const DELETE_MESSAGE_BODY = 'Todo deleted successfully';

export const TODO_COMPLETED_MESSAGE_TITLE = 'Congratulations!';
export const TODO_COMPLETED_MESSAGE_BODY = 'Todo completed successfully';
```

Then, update the `createNotification` method in `todoApp.ui.js` to use these new constants. For example:

```javascript
case 'success':
  NotificationManager.success(SUCCESS_MESSAGE_BODY, SUCCESS_MESSAGE_TITLE);
  break;
```